app/views/projects/_import_export.html.erb

```diff
@@ -1,13 +1,23 @@
 <details id="import-export">
   <summary class="import-export-header"">
     <i class="fa fa-lg fa-chevron-down rotate"></i>
-    <h4 class="import-export-header__text">CSV Import / Export</h4> </summary>
+    <h4 class="import-export-header__text">CSV Import / Export</h4>
+  </summary>
   <section>
     <div id="import" class="card-deck mb-3">
       <div class="card mb-6 box-shadow">
         <div class="card-header">
           <h4 class="my-0 font-weight-normal">Import CSV</h4>
         </div>
+        <div id="instructions">
+          <span>Instructions:</span>
+          <ul class="list">
+            <li class="list-item">The file must have the following columns and headers: id, title, description and position.</li>
+            <li class="list-item">The column title is required.</li>
```

Comment:
Maybe say `The "title" column is required.`? I read that quickly and thought `column title` referred to the column's header 😅

Reply:
Looks like `description` and `position` are also required https://github.com/fastruby/points/blob/main/app/controllers/stories_controller.rb#L126